chips/cc26x2/src/radio/rfc.rs

```diff
@@ -118,6 +118,19 @@ register_bitfields! {
     ]
 }
 
+#[allow(unused)]
+macro_rules! rfc_cmd_ack_nvic {
```

Comment:
Unused? If you are going to keep this, I wonder if you need it as a macro?

Reply:
forgot this was still in here due to the allow hiding it. Will remove